### DISCUSSION TITLE
一点想法

### DISCUSSION THREAD

**Discussion by neoedmund** [jynew/jynew#1058] [Ideas]
虽然最不缺的就是想法，但是还是想说一下。没有贬意，也算自勉。

同人游戏没前途。既然花了那么多精力，为什么不搞个原创的呢?

看到"耻辱榜"，感叹这些国内公司好没有底线，就随意剽窃。但是回头一想，这个项目本来就是剽窃(重制)啊。只不过技术含量不同。

但是真的，如果是原创，排除掉版权有关的任务，剧情，玩法。是不是完全是野鸡变凤凰一样的蜕变呢?

**Comment by qige2016** [comment]
想法一样，计划把整个框架抽离一个独立的引擎出来，金群3D只是它的一个mod，目前全力开发本体游戏《渡城残魂传》，验证这个框架的扩展性。

**Comment by qige2016** [comment]
有兴趣可以看看https://www.bilibili.com/video/BV1jS4y1E7n5/

**Reply by neoedmund** [reply]
哈哈 你这音乐就说明你没真正明白我说的意思。

**Reply by MephistoSatanDevil** [reply]
> 哈哈 你这音乐就说明你没真正明白我说的意思。

因为还没做完。又没人写音乐，总不能全程无声吧。要换的早晚会换。

**Comment by hwtlittleming** [comment]
抽离出框架后，游戏里的3dmodel和动作是没版权的，也得替换吧

**Reply by MephistoSatanDevil** [reply]
模型和动作不存在这个问题。
